subgraphs/radiant-capital/README.md

```diff
@@ -0,0 +1,171 @@
+
+-  Loop function allows users to increase their RDNT token rewards and earn yield on a larger collateral value 
+
+### Vesting and Locking RDNT
+
+#### Vesting RDNT
+- Users vest their RDNT tokens to receive 50% of all fee-generated (borrowing interest paid) while actively vesting. 
+
+- Users can withdraw their vested tokens for a 50% penalty on the RNDT rewards.
+
+- The 50% penalty of RDNT rewards are distributed to users who lock their RDNT tokens
+
+##### Locking RDNT
+- Users who lock their RDNT tokens receive 50% of fee generated as well. 
+
+- They will also receive additional income from RDNT penalty fees from all positions who exit their vests early. 
+
+- Users who lock their tokens can unlock but cannot fully withdraw them.
+
+### Pool2 
+- Liquidity providers of RDNT/WETH pair on SushiSwap can stake their LP tokens in Radiant to receive rewards in the form of RDNT tokens 
+- This pool aims to enable deeper liquidity for the protocol to secure the longevity and health of Radiant platform
+
+## Useful Links
+Protocol:
+- https://app.radiant.capital/#/markets 
+
+Docs:
+- https://docs.radiant.capital/docs/ 
+
+Smart contracts:
+- https://docs.radiant.capital/docs/contracts-and-security/contracts 
+
+Discord:
+- https://discord.com/invite/radiantcapital
+
+## User Metrics
+The user actions of Radiant include:
+
+### Lending and Borrowing
+#### Lending
+- Deposit any of the five supported assets such as WBTC, WETH, USDT, USDC, & DAI in exchange of rTokens
+- Claim RDNT rewards from the deposited assets
+
+#### Borrowing
+- Borrow after depositing assets as collaterals
+- Repay borrowed funds
+- Claim RNDT rewards from the borrowed assets
+
+#### Liquidation
+- Liquidate undercollateralized positions/loans
+
+### 1-Click Looping
+- Automate and repeat the process of borrow and deposit cycle multiple times by looping to earn greater yield
+
+### Vesting and Locking RDNT tokens
+#### Vesting
+- Vest RDNT tokens 
+- Users who vest RDNT may choose to exit early within 28 days (four weeks).
+- Claim RDNT rewards incurred from vested tokens
+
+#### Locking
+- Lock RDNT tokens to get rewards
+- Claim rewards from the locked tokens
+- Unlock RDNT tokens
+
+### Pool2
+- Staking RDNT/WETH LP tokens from SushiSwap in Radiant platform
+- Vest or lock your earned RDNT rewards from the pool
+
+## Financial Metrics
+### TVL
+
+> TVL of a Single Pool = $\sum$ value of the specific token deposited in the contract 
+
+> TVL of a Protocol = $\sum$  TVL of single pools 
+
+Note: At the time of writing, the platform only supports assets or tokens such as BTC, ETH, USDC, USDT, and DAI.
+
+And,
+
+> TVL of Pool2 = $\sum$  value of LP tokens of RDNT/WETH from SushiSwap staked in Radiant platform
+
+Note: This refers to RDNT/WETH liquidity pool on SushiSwap and staking the LP token in Radiant to earn more RDNT.
+
+### Total Revenue
+> Total Revenue = $\sum$  value of interest payments incurred by borrowers (platform fees)
```

Comment:
Liquidation fees should be left out of total revenue?

Reply:
Liquidation is not revenue. Revenue is only interest paid.

Reply:
@ishraq8 just wanted to confirm with you here, do we include liquidation as revenue?

Reply:
We don not include liquidations as revenue. It makes it more complicated. Radiant looks like an AAVE/COMP fork on arbitrum, we don't include liquidations as revenue on those protocols either. The revenue from liquidations go to liquidators using a `liquidationCall()` function ([docs](https://docs.radiant.capital/docs/project-info/borrow/liquidations). It is not distributed to suppliers.

Reply:
Sounds good. Thanks @ishraq8!